src/online.js

```diff
@@ -403,11 +403,11 @@ async function showTabs() {
   myPlayerId = authClient.playerId;
   setMyName();
 
-  const avatar = (await gameClient.getPlayersAvatar([ myPlayerId ]))[0];
+  const avatar = (await gameClient.getPlayersAvatar([ myPlayerId ]))[0]; 
 
   document.body.classList.toggle('account-is-at-risk', isAccountAtRisk);
 
-  await avatarsPromise;
+  await getDataFromService();
```

Comment:
This is the cause of your bug.  The original code awaits a shared promise object.  That promise object might already be resolved or it might not.  But the new code creates a new promise and performs redundant operations.